### ISSUE TITLE
Feature Request: Add count notification to subscriptions

### ISSUE THREAD

**Issue by aproposnix** [FreeTubeApp/FreeTube#1]
It would be nice to see an unwatched number notification when scrolling through my subscription (on the left panel) list.  A typical bubble icon would work perfectly. 

This is a really nice app.  Keep up the great work.  It may replace minitube for me :)

**Comment by PrestonN** [comment]
I'll have to see what I can do about this.  A feature like this would have to keep up with the last time you loaded your subscriptions.  Depending on the amount of subscribers a user has and with the last time the user has used the application, this could cause a ton of API requests, which would slow down the application as a whole.

Having said that, I think a simple solution could just be a dot that states if that channel has new uploads compared to the last time you opened FreeTube.  This will be less descriptive but it should still be informative.

Let me know what you think.

**Comment by aproposnix** [comment]
> A feature like this would have to keep up with the last time you loaded your subscriptions. Depending on the amount of subscribers a user has and with the last time the user has used the application, this could cause a ton of API requests, which would slow down the application as a whole.

Couldn't this be done client side by utilizing the history a little more?  I mean, currently the clicked videos are added to the history (which I assume is added to the local DB). If you add scrolled through (like auto-read on an RSS feed reader) you would then have the full history. So then, for each subscription it's a check against the history to see which was the last video watched or scrolled through (both counting a seen) anything newer counts as unread.

**Comment by PrestonN** [comment]
You are correct that the history is located in a local DB.  I'm not sure if I would want to add videos that are only scrolled through into the history DB.  We could make a seperate database that stores this in an effort to prevent the current history database from cluttering up.

**Comment by aproposnix** [comment]
Well, using the method described allows you to filter the history.  I think by default you would just display 'watched' videos. There could then be a tickbox for showing 'all' history including the 'browsed' videos. In order to mitigate the DB from getting stupidly large, you could allow the user to trim the history (should probably be done by default). You could even allow separate trimming options for watched and browsed.

**Comment by Dreadaxe** [comment]
Possibility to have configure notifications to Windows notification bar

**Comment by Menjuel** [comment]
Are there any news on this? I can't see any notification dots on my subscriptions when there is a new video. Is there a setting I am missing? I am on Manjaro.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @Menjuel this isn't implemented and it seems that https://github.com/FreeTubeApp/FreeTube/issues/546 is similar to this one.

One solution could be is to do it like Newpipe does. On subscription refresh it will show u what videos are new with a border around the new videos. If u close newpipe an reopen the border still stays there but if u refresh the border goes away. Except when there are new videos of course.

**Comment by sentix001** [comment]
PLEASE ADD NOTIFICATIONS FOR SUBSCRIBED VIDEO

**Comment by sentix001** [comment]
GOOGLE IS ALREADY TESTING OUT BLOCKING ADBLOCKERS SO THIS WILL BE THE NEW SALVATION FOR ALL
